Userland/Games/Snake/Game.h

```diff
@@ -77,8 +99,8 @@ class Game : public GUI::Frame {
     bool m_is_new_high_score { false };
 
     NonnullRefPtrVector<Gfx::Bitmap> m_food_bitmaps;
-
-    Gfx::Color m_snake_base_color { Color::Yellow };
+    NonnullRefPtrVector<Gfx::Bitmap> m_snake_bitmaps;
+    NonnullRefPtrVector<Gfx::Bitmap> m_head_bitmaps;
```

Comment:
Each of these always has a fixed number of elements. Maybe a `FixedArray` would be better.

Reply:
This would complicate the code severely and isn't important compared to the pain and suffering that it would take.